scripts/safe_template_linter.py

```diff
@@ -180,6 +185,34 @@ def __init__(self, rule):
         """
         self.rule = rule
         self.full_path = ''
+        self.is_disabled = False
+
+    def _mark_disabled(self, template_string):
+        """
+        Performs the disable pragma search and marks the rule as disabled if a
+        matching pragma is found.
+
+        Pragma format::
+
+            safe-lint: disable=violation-name,other-violation-name
+
+        Arguments:
+            template_string: The scoped string of code in which to search for
+            the pragma.
+
+        Side Effect:
+            Sets self.is_disabled as appropriate based on whether the pragma is
+            found.
+
+        """
+        pragma_match = re.search(r'safe-lint:[\s*]disable=([a-zA-Z-,]+)', template_string)
```

Comment:
`[\s*]` isn't what you want here, that will look for either a whitespace, or a star, and only one.  Just remove the square brackets.